pkg/codegen/docs/templates/resource.tmpl

```diff
@@ -14,8 +14,32 @@
 {{- end }}
 
 <!-- Create resource -->
-## Create {{ .Header.Title }} Resource {#create}
+{{ if ne (len .CreationExampleSyntax) 0 }}## Creating a {{ .Header.Title }} Resource
```

Comment:
small nit: should we revert this back to how the title it was before, i.e. Create <resource title> Resource? I notice sometimes this turns up grammatically incorrect when the resource name begins with a vowel, e.g. "Create a Instance Resource".

Reply:
I went ahead and reverted it back to the original heading. Feel free to let me know if you have any other ideas or push back of course 🙂